apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/DiagnosisUtilV2.java

```diff
@@ -564,7 +564,7 @@ static int addDiagnosisCode(ExplanationOfBenefit eob, Diagnosis diagnosis, Claim
    */
   static Optional<Integer> addDiagnosisCode(
       ExplanationOfBenefit eob, Optional<Diagnosis> diagnosis, ClaimType claimType) {
-    return diagnosis.map(d -> new Integer(addDiagnosisCode(eob, d, claimType)));
+    return diagnosis.map(d -> (addDiagnosisCode(eob, d, claimType)));
```

Comment:
was this just unneeded code unrelated to the change?

Reply:
Yes. This was not related to the code. We were getting a warning abou the deprecation of the Integer(int). So just took it out since it wasnt needed.